cms/static/js/views/content_group_list.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * This class defines a list view for content groups.
+ * It is expected to be backed by a Group collection.
+ */
+define([
+    'js/views/list', 'js/views/content_group_item', 'gettext'
+], function(ListView, ContentGroupItemView, gettext) {
+    'use strict';
+
+    var ContentGroupListView = ListView.extend({
+        tagName: 'div',
+
+        className: 'group-configurations-list',
```

Comment:
I'm not sure, but good catch. I'll fix this up during my rebasing (I hope it isn't part of a larger rebase issue).